torntools/scripts/content/profile/ttProfile.js

```diff
@@ -395,6 +396,8 @@ function profileLoaded() {
 }
 
 function displayAlly(user_faction, allies) {
+	if (userId == userdata.player_id) return;
```

Comment:
Don't add this check here. Just move the `displayAlly()` callout under the line `if (userId == userdata.player_id) return;` (#224 in original file)